src/posix/platform/hdlc_interface.cpp

```diff
@@ -247,6 +249,94 @@ otError HdlcInterface::Write(const uint8_t *aFrame, uint16_t aLength)
     return error;
 }
 
+otError HdlcInterface::WaitResponse(struct timeval &aTimeout)
```

Comment:
Is it possible to divide this into `WaitReadable()` and `Read()`? Such that `WaitResponse()` can be implemented in RadioSpinel.

Reply:
Yes. We can divide it into ` WaitReadable()` and `Read()`. But if we do it like this, it increases an API between RadioSpinel and HdlcInterface and reduces the flexibility of the low layer.

Reply:
As discussed, we should rename this method, because request/response is spinel level concepts.

Reply:
Updated.